db/fixtures/ae_datastore/ManageIQ/System/CommonMethods/QuotaMethods.class/__methods__/limits.rb

```diff
@@ -42,12 +42,21 @@ def vmdb_object(object, id)
 def parent_model_value(attr)
   value = $evm.parent[attr].to_i
   $evm.log(:info, "Quota Model #{attr}: #{value}") unless value.zero?
+  if attr.ends_with?("storage") || attr.ends_with?("memory")
+    value *= 1024**2
```

Comment:
@billfitzgerald0120 
Rails has these functions which make it simpler to convert to byes
value.megabytes          would take megabyte value and convert it to bytes
value.gigabytes            would take a gigabyte value and convert it to bytes